ddtrace/tracer/textmap.go

```diff
@@ -55,7 +55,7 @@ func (c TextMapCarrier) ForeachKey(handler func(key, val string) error) error {
 }
 
 const (
-	defaultBaggageHeaderPrefix = "ot-baggage-"
+	defaultBaggageHeaderPrefix = "x-baggage-"
```

Comment:
@gbbr probably we need to keep `ot-baggage-` since it has been used also in other tracers (ref: https://github.com/DataDog/dd-trace-java/blob/a2010ef4a4b7cb7fc7f095a34902342b0b90147a/dd-trace-ot/src/main/java/datadog/opentracing/propagation/HTTPCodec.java#L18)